### PR TITLE
destroy instances - server logs

### PR DESCRIPTION
**Pull request by Shellyber**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/30291

## Screenshots
image
image

### CODE REVIEW

**Comment by Shellyber** [comment]
Ready and works. Now it's adding all server logs.
Nightly build for example: https://app.circleci.com/pipelines/github/demisto/content/42083/workflows/a94afdc6-9de0-4b32-83be-f24b5abed9e6/jobs/181834

File: Tests/scripts/destroy_instances.py [hunk 1]
```diff
@@ -16,18 +18,19 @@ def main():
...
                      '{}@{}:/var/log/demisto/server.log {} || echo "WARN: Failed downloading server.log"'
 
         try:
+            logging.debug(f'Trying to run {ssh_string}')
             subprocess.check_output(
                 ssh_string.format(env["SSHuser"], env["InstanceDNS"]), shell=True)
 
         except subprocess.CalledProcessError as exc:
-            print(exc.output)
+            logging.exception(exc.output)
```
Comment (hod-alpert) [inline Tests/scripts/destroy_instances.py:33]:
```suggestion
            logging.exception(f'Failed changing permissions of folder /var/log/demisto on server {env["InstanceDNS"]}')
```

File: Tests/scripts/destroy_instances.py [hunk 2]
```diff
@@ -16,18 +18,19 @@ def main():
...
     for env in filtered_results:
-        print(f'Downloading server log from {env.get("Role", "Unknown role")}')
+        logging.info(f'Downloading server log from {env.get("Role", "Unknown role")}')
         ssh_string = 'ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null {}@{} ' \
                      '"sudo chmod -R 755 /var/log/demisto"'
         scp_string = 'scp -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null ' \
                      '{}@{}:/var/log/demisto/server.log {} || echo "WARN: Failed downloading server.log"'
 
         try:
+            logging.debug(f'Trying to run {ssh_string}')
```
Comment (hod-alpert) [inline Tests/scripts/destroy_instances.py:28]:
```suggestion
            logging.debug(f'Changing permissions of folder /var/log/demisto on server {env["InstanceDNS"]}')
```

File: Tests/scripts/destroy_instances.py [hunk 3]
```diff
@@ -16,18 +18,19 @@ def main():
...
             subprocess.check_output(
                 ssh_string.format(env["SSHuser"], env["InstanceDNS"]), shell=True)
 
         except subprocess.CalledProcessError as exc:
-            print(exc.output)
+            logging.exception(exc.output)
```
Comment (hod-alpert) [inline Tests/scripts/destroy_instances.py:34]:
```suggestion
logging.debug(f'Downloading server logs from server {env["InstanceDNS"]}')
```

File: Tests/scripts/destroy_instances.py [hunk 4]
```diff
@@ -39,18 +42,18 @@ def main():
                 shell=True)
 
         except subprocess.CalledProcessError as exc:
-            print(exc.output)
+            logging.exception(exc.output)
```
Comment (hod-alpert) [inline Tests/scripts/destroy_instances.py:45]:
```suggestion
            logging.exception(f'Failed downloading server logs from server {env["InstanceDNS"]}')
```

File: Tests/scripts/destroy_instances.py [hunk 5]
```diff
@@ -39,18 +42,18 @@ def main():
...
-            return
+            logging.info(f'Skipping - Time to live was set to {time_to_live} minutes')
+            continue
         if os.path.isfile("./Tests/is_build_passed_{}.txt".format(env["Role"].replace(' ', ''))):
-            print(f'Destroying instance {env.get("Role", "Unknown role")}')
+            logging.info(f'Destroying instance {env.get("Role", "Unknown role")}')
             rminstance = aws_functions.destroy_instance(env["Region"], env["InstanceID"])
             if aws_functions.isError(rminstance):
-                print_error(rminstance)
+                logging.error(rminstance)
```
Comment (hod-alpert) [inline Tests/scripts/destroy_instances.py:54]:
```suggestion
                logging.error(rminstance["Message"])
```

File: Tests/scripts/destroy_instances.py [hunk 6]
```diff
@@ -39,18 +42,18 @@ def main():
...
+            logging.exception(exc.output)
 
         if time_to_live:
-            print(f'Skipping - Time to live was set to {time_to_live} minutes')
-            return
+            logging.info(f'Skipping - Time to live was set to {time_to_live} minutes')
+            continue
         if os.path.isfile("./Tests/is_build_passed_{}.txt".format(env["Role"].replace(' ', ''))):
-            print(f'Destroying instance {env.get("Role", "Unknown role")}')
+            logging.info(f'Destroying instance {env.get("Role", "Unknown role")}')
```
Comment (hod-alpert) [inline Tests/scripts/destroy_instances.py:51]:
```suggestion
            logging.info(f'Destroying instance with role - {env.get("Role", "Unknown role")} and IP - 
 {env["InstanceDNS"]}')
```

**Review by hod-alpert — APPROVED** [review]
Looks good, just change the logs a bit

**Pull request merged**